packages/paste-website/src/pages/components/keyboard-key/index.mdx

```diff
@@ -0,0 +1,413 @@
+
+When writing out keys:
+
+<List marginBottom="space60">
+  <ListItem>Use title case. Example: “Caps Lock”, not “Caps lock”.</ListItem>
+  <ListItem>
+    For modifier keys like Control, Command, and Option, spell out the key instead of using abbreviations or symbols.
+    Example: “Control”, not “Ctrl” or “^”. Use the abbreviation only when space is limited.
+  </ListItem>
+  <ListItem>Use “Enter” instead of “Return”.</ListItem>
+</List>
+
+### Common keyboard shortcuts
+
+Common platform-level shortcuts that are used across Twilio include:
+
+<Box marginBottom="space60">
+  <Table>
+    <THead>
+      <Tr>
+        <Th>Action</Th>
+        <Th>Windows/Linux</Th>
+        <Th>Mac</Th>
+      </Tr>
+    </THead>
+    <TBody>
+      <Tr>
+        <Td>Open search</Td>
+        <Td>
+          <KeyboardKeyGroup>
+            <KeyboardKey>Control</KeyboardKey>
+            <KeyboardKey>K</KeyboardKey>
+          </KeyboardKeyGroup>
+        </Td>
+        <Td>
+          <KeyboardKeyGroup>
+            <KeyboardKey>Command</KeyboardKey>
+            <KeyboardKey>K</KeyboardKey>
+          </KeyboardKeyGroup>
+        </Td>
+      </Tr>
+      <Tr>
+        <Td>Bold text</Td>
+        <Td>
+          <KeyboardKeyGroup>
+            <KeyboardKey>Control</KeyboardKey>
+            <KeyboardKey>B</KeyboardKey>
+          </KeyboardKeyGroup>
+        </Td>
+        <Td>
+          <KeyboardKeyGroup>
+            <KeyboardKey>Command</KeyboardKey>
+            <KeyboardKey>B</KeyboardKey>
+          </KeyboardKeyGroup>
+        </Td>
+      </Tr>
+      <Tr>
+        <Td>Italicize text</Td>
+        <Td>
+          <KeyboardKeyGroup>
+            <KeyboardKey>Control</KeyboardKey>
+            <KeyboardKey>I</KeyboardKey>
+          </KeyboardKeyGroup>
+        </Td>
+        <Td>
+          <KeyboardKeyGroup>
+            <KeyboardKey>Command</KeyboardKey>
+            <KeyboardKey>I</KeyboardKey>
+          </KeyboardKeyGroup>
+        </Td>
+      </Tr>
+      <Tr>
+        <Td>Underline text</Td>
+        <Td>
+          <KeyboardKeyGroup>
+            <KeyboardKey>Control</KeyboardKey>
+            <KeyboardKey>U</KeyboardKey>
+          </KeyboardKeyGroup>
+        </Td>
+        <Td>
+          <KeyboardKeyGroup>
+            <KeyboardKey>Command</KeyboardKey>
+            <KeyboardKey>U</KeyboardKey>
+          </KeyboardKeyGroup>
+        </Td>
+      </Tr>
+    </TBody>
+  </Table>
+</Box>
+
+From [W3C's guide to developing a keyboard interface](https://www.w3.org/WAI/ARIA/apg/practices/keyboard-interface/#keyassignmentconventionsforcommonfunctions):
+The following keyboard commands should be used in any context where the actions are appropriate.
```

Comment:
suggestion: can you clarify this wording a bit? it's a little confusing what you're recommending

Reply:
I think Sarah was trying to say that the following list of keyboard shortcuts can be used where the associated action is appropriate. So command+x can be used but only as a cut action. The paragraph below this puts it into context.